### ISSUE TITLE
allow arbitrary additional kernel options

### ISSUE THREAD

**Issue by bmr-cymru** [bmr-cymru/boom#16]
It should be possible to specify arbitrary "extra" options to append to the kernel command line, for debugging, customisation or other purposes:

```
# boom create --append noacpi
```

**Comment by bmr-cymru** [comment]
Thinking about this some more, it would also be useful to be able to selectively mask/delete existing templated options, for e.g.:

```
# boom create --delete "rhgb quiet"
# boom create --mask rhgb,quiet
```

Not sure on the best form for the command line option yet but this is fairly straightforward to do.

**Comment by bmr-cymru** [comment]
Append work is on branch `bmr-boom-append`: appending options now works, and they are stored with the BootEntry on-disk and recovered during BootParams reconstruction (this means that they will survive manual edits to the file: any options not recognised by the template reversing will be considered to be appended opts and added to the append list).

**Comment by bmr-cymru** [comment]
Final syntax:

```
# boom create --title foo --rootlv vg_hex/root --add-opts debug --del-opts "rhgb quiet"
```

Closed in commit 1b88d8d1a07ed387bd387900f7ed90928341905e.
